sdk/monitor/Azure.Monitor.Query/src/MetricsQueryClient.cs

```diff
@@ -24,7 +24,9 @@ public class MetricsQueryClient
         private readonly ClientDiagnostics _clientDiagnostics;
 
         /// <summary>
-        /// Initializes a new instance of <see cref="MetricsQueryClient"/>. Uses the default 'https://management.azure.com' endpoint.
+        /// Creates an instance of <see cref="MetricsQueryClient"/> for Azure Public Cloud usage. Uses the default 'https://management.azure.com' endpoint.
+        /// To access an Azure sovereign cloud, use the following constructor overload:
+        /// <see cref="MetricsQueryClient.MetricsQueryClient(TokenCredential, MetricsQueryClientOptions)"/>
```

Comment:
Should this be `MetricsQueryClient(Uri, TokenCredential, MetricsQueryClientOptions)`? We needed to include the ResourceManagerEndpoint to be that of a sovereign cloud, i.e., https://github.com/Azure/azure-sdk-for-net/blob/efcd23021b748f3de1b98c20b9f430490245574d/sdk/resourcemanager/Azure.ResourceManager/src/ArmEnvironment.cs#L24

<img width="535" alt="image" src="https://github.com/Azure/azure-sdk-for-net/assets/25066231/f84cb1bb-2b4a-4574-9ac0-580747005345">

Reply:
When the v1.3.0 release ships later this month, we'll recommend the ctor overload I referenced in the XML docs.